params.cpp

```diff
@@ -185,6 +185,8 @@ bool Params::parseParams(const int argc, const char *argv[]) {
     QStringList params;
     for (int i = 1; i < argc; i++) {
         params.push_back(argv[i]);
+        QuasarAppUtils::Params::log(QString("Arguments[%0]: %1").arg(i).arg(argv[i]),
+                                    QuasarAppUtils::Info);
```

Comment:
Hm i thin that you need to change the QuasarAppUtils::Info  to the QuasarAppUtils::Debug.
Because it will be very cluttering in the output of applications.